src/main/java/com/google/api/codegen/transformer/TestCaseTransformer.java

```diff
@@ -67,7 +67,11 @@ public TestCaseView createTestCaseView(
               methodContext.getTypeTable(),
               methodConfig.getPageStreaming().getResourcesFieldConfig());
     } else if (methodConfig.isLongRunningOperation()) {
-      clientMethodName = namer.getAsyncApiMethodName(method, methodConfig.getVisibility());
+      if (clientMethodType == ClientMethodType.OperationOptionalArrayMethod) {
+        clientMethodName = namer.getApiMethodName(method, methodConfig.getVisibility());
+      } else {
+        clientMethodName = namer.getAsyncApiMethodName(method, methodConfig.getVisibility());
```

Comment:
I am not sure what the purpose of this else case. It looks confusing to me that the method name is "async" while it is LRO. Do we want to distinguish those two types of methods?